patches/api/0085-Add-PlayerAdvancementCriterionGrantEvent.patch

```diff
@@ -58,6 +63,17 @@ index 0000000000000000000000000000000000000000..bb8d7c959cdea4b66455a49e74804ea4
 +        return cancel;
 +    }
 +
++    /**
++     * Gets the current AdvancementProgress.
```

Comment:
thing is if we do say it returns the current one. it actually has to return the current one which it doesn't since the advancement progress is being retrieved during event creation and not during the method call. i am unsure if the jdocs or the way its stored is the issue. somebody from paper team should probably look over that one and say how its usually done with events

Reply:
might be fine though. probably just me being dyslexic

Reply:
From what I can tell, anything that would invalidate the advancement progress would also invalidate the advancement itself that is provided in the same way, so I thought it was unnecessary

Reply:
yeah good point. LGTM then